pkgs/tools/misc/git-town/default.nix

```diff
@@ -0,0 +1,18 @@
+let 
+  sha_hash = (import ./sha.nix);
+  version = (import ./version.nix);
+in
+
+{ stdenv, fetchurl, go}:
+
+  stdenv.mkDerivation { 
```

Comment:
please use buildGoPackage instead of custom builder. https://nixos.org/nixpkgs/manual/#sec-language-go
This enable all sorts of niceties, see https://github.com/NixOS/nixpkgs/issues/52469

Reply:
Changed to use buildGoPackage